.travis.yml

```diff
@@ -1,13 +1,76 @@
 ---
 sudo: false
-language: csharp
-solution: MaxMind.Db.sln
+language: c++
+# dotnet cli require Ubuntu 14.04
+sudo: required
+dist: trusty
+
+# dotnet cli require OSX 10.10
+osx_image: xcode7.1
+
+addons:
+  apt:
+    packages:
+    - gettext
+    - libcurl4-openssl-dev
+    - libicu-dev
+    - libssl-dev
+    - libunwind8
+    - zlib1g
+
+os:
+  - osx
+  - linux
+
+env:
+  matrix:
+    - CLI_VERSION: Latest
+      Configuration: Debug
+      Framework: netcoreapp1.0
+    - CLI_VERSION: Latest
+      Configuration: Release
+      Framework: netcoreapp1.0
+
 before_install:
-  - git submodule update --init --recursive
-  - nuget install NUnit.Runners -Version 2.6.3 -o packages
+  # Download script to install dotnet cli
+  - curl -L --create-dirs https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/install.sh -o ./scripts/install.sh
+
+  - find ./scripts -name "*.sh" -exec chmod +x {} \;
```

Comment:
I don't think this is necessary given that you are passing the script to bash as an argument.